src/converse-core.js

```diff
@@ -335,6 +335,7 @@
             sid: undefined,
             sounds_path: '/sounds/',
             storage: 'session',
+            synchronize_presence: true, // Set to false to not sync with other clients or with resource name of the particular client that it should synchronize with
             use_vcards: true,
```

Comment:
Actually, there might be a whole different way to fix that problem. Instances of converse.js in the same browser (in different tabs) all share the same session storage as far as I'm aware. Each instance however stores it's data under a unique key, so they don't read one another's data. In this case however, we'd like to know if another instance has changed the status. I think what could be done, would be to store the user's status in a global in sessionStorage, for example under the bare JID. Then when one instance changes it, other instances should immediately know about it, since it should change for them as well, since they share the sessionStorage. That's the theory at least.